backends/graphics3d/openglsdl/openglsdl-graphics3d.cpp

```diff
@@ -563,7 +580,7 @@ bool OpenGLSdlGraphics3dManager::isVSyncEnabled() const {
 void OpenGLSdlGraphics3dManager::drawOverlay() {
 	_surfaceRenderer->prepareState();
 
-	glViewport(0, 0, _overlayScreen->getWidth(), _overlayScreen->getHeight());
+	glViewport(_overlayDrawRect.left, _windowHeight - _overlayDrawRect.top - _overlayDrawRect.height(), _overlayDrawRect.width(), _overlayDrawRect.height());
```

Comment:
Little nitpick: this could be `_overlayDrawRect.bottom`

Reply:
This looks to be less readable than the original code, though, and the optimization done won't make such a difference, IMHO